jovian/utils/clone.py

```diff
@@ -97,8 +98,10 @@ def clone(slug, version=None, fresh=True, include_outputs=True):
             set_notebook_slug(f['filename'], slug)
 
     # Print success message and instructions
-    if fresh:
+    if fresh and not overwrite:
         post_clone_msg(title)
+    elif overwrite:
+        log(f'Downloaded files into {title}')
```

Comment:
This message can be improved.. Any suggestions? @aakashns @siddhantujjain

Reply:
No need to change the message here. `post_clone_msg` should do